docs/learn/learn-extrinsics.md

```diff
@@ -0,0 +1,73 @@
+
+## Extrinsics (a.k.a. Types of Transactions)
+
+Now that we introduced where the term _extrisic_ comes from, we can explain what extrinsics really are. In Substrate, a _"thing that could exist in the transaction pool"_ — more commonly known as "a transaction" in the non-Substrate world — can be one of 3 distinct types, all of which fall under a broader category called "extrinsics". Extrinsics is just a general term to mean "any information that originates from outside a runtime". These are:
+
+- **Signed transactions:** these must contain the signature of the account sending the inbound request to the runtime. With signed transactions, the account used to submit the request typically pays a transaction fee and must sign it using the account's private key.
+- **Unsigned transactions:** these don't carry any information about who submitted the transaction, since the format of this type of transaction doesn't require a signature. You can define what conditions must be met for such a transaction to be valid.
+- **Inherents:** are a special type of unsigned transaction made by block authors which carry information required to build a block such as timestamps, storage proofs and uncle blocks.
+ 
+Here are some key differences between the different types of extrinsics:
+
+- Contrary to signed transactions, unsigned transaction types require implementing custom validation logic which can consume more resources for checking validity compared to signed transactions.
+- Unsigned transactions have no economic deterrent to prevent spam or replay attacks, so custom logic must account for protecting the network from these types of transactions being misused.
+- Inherents exist to address the need of adding some data to a block, whereas signed or unsigned transactions exist to potentially change the state of the blockchain.
+
+## Importance of Verifying Extrinsics
+
+:::info Walk-through Video Tutorial 
+
+Visit the [dedicated support page](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-) and see [this technical explainer](https://youtu.be/bxMs-9fBtFk) tutorial to learn about how to verify extrinsics before signing them. The video will also mention potential attacks that can happen to you while signing for transactions. **Do not sign any transaction if you suspect you are or have been under attack.**
+
+:::
+
+:::warning Always check for Metadata Updates
+
+Before signing extrinsics with the Polkadot-JS Browser Extension and Parity Signer, always check for metadata updates. [This technical explainer video](https://youtu.be/gbvrHzr4EDY) will explain how to do it. Failing to update metadata will result in errors while signing that for example will ultimately result in you not being able to transfer funds. Before authorizing the metadata update check who is requesting it. Any request not coming from the Polkadot-JS Apps must not be signed.
```

Comment:
Metadata updates for Signer and the extension are two different things, of various reasons. 

- Signer gets the full metadata on the device through the QR fountain. The extension updates the metadata index; the metadata isn't loaded onto it (contrary to what I had told you in the past).
- The process of updating the metadata is quite different as a result
- Having outdated metadata on Signer will prevent you from signing. On the extension you'll probably be able to sign but the extrinsic might fail (never tried it though tbh).

So, I think this section needs a bit of rewriting.

Also, this:
> Any request not coming from the Polkadot-JS Apps must not be signed.

is not correct. dApps may request a metadata update too (for example DeFi apps of parachains). So, for the extension, you should trust the app that requests the update. For Signer, you should trust the issuer of the metadata (or do it yourself).

Reply:
Content corrected and updated.